packages/repository-json-schema/src/__tests__/unit/build-schema.unit.ts

```diff
@@ -232,13 +232,27 @@ describe('build-schema', () => {
       expect(key).to.equal('modelPartial');
     });
 
-    it('returns concatenated option names otherwise', () => {
+    it('returns "optional[id,_rev]" when a single option "optional" is set', () => {
+      const key = buildModelCacheKey({optional: ['id', '_rev']});
+      expect(key).to.equal('modelOptional[id,_rev]');
+    });
+
+    it('does not include "optional" in concatenated option names if it is empty', () => {
       const key = buildModelCacheKey({
-        // important: object keys are defined in reverse order
```

Comment:
Please preserve this comment in the test `it('returns concatenated option names otherwise', () => {`